examples/bq-remote-function/README.md

```diff
@@ -0,0 +1,18 @@
+# BigQuery Remote Function Java Samples
```

Comment:
You mention samples here, but it is one sample. Are you planning on adding more samples in the future?